tests/snuba/search/test_backend.py

```diff
@@ -2492,6 +2492,26 @@ def test_cursor_performance_issues(self):
         assert results.hits == 2
 
     def test_not_handled_function(self):
+        with self.feature("organizations:issue-platform"):
```

Comment:
Do you mind adding a quick docstring to these tests just saying they're testing unsupported queries?